### DISCUSSION TITLE
Unidirectional Routing

### DISCUSSION THREAD

**Discussion by davidcarails** [The-OpenROAD-Project/OpenROAD#2115] [Q&A] [answered]
Is there a way to get unidirectional routing?

**Comment by maliberty** [comment, marked as answer]
If your LEF has a RECTONLY constraint or multiple masks then you will get unidirectional routing.  We don't have a direct user knob for that.  You can also create the constraint directly using the odb APIs.

**Reply by davidcarails** [reply]
Thank you
